pkg/core/annotations.go

```diff
@@ -35,3 +37,39 @@ func (c *client) bumpNonceAnnotation(s *v1alpha1.Service) {
 	annotations[buildAnnotation] = strconv.Itoa(i + 1)
 	s.Spec.RunLatest.Configuration.RevisionTemplate.SetAnnotations(annotations)
 }
+
+func (c *client) bumpNonceAnnotationForBuild(s *v1alpha1.Service) {
+	configurationSpec := s.Spec.RunLatest.Configuration
+	build := getBuild(configurationSpec)
+	if build != nil {
+		annotations := build.GetAnnotations()
```

Comment:
There's no need to use the get/set method, you can interact with the Annotations field directly.

Reply:
Hmmm, just keeping the same code we already had, also not seeing how I can manipulate the annotations directly on an Unstructured object